ruby/vampires.rb

```diff
@@ -1,10 +1,80 @@
-puts "What is your name?"
-vampire_name = gets.chomp
-puts "How old are you?"
-age = gets.chomp
-puts "What year were you born?"
-birth_year = gets.chomp
-puts "Our company cafeteria serves garlic bread. Should we order some for you?"
-garlic = gets.chomp
-puts "Would you like to enroll in the company's health insurance?"
-insurance = gets.chomp
\ No newline at end of file
+# Method to convert yes/no responses to booleans
+
+def boolean_convert(str)
+  if str == "yes"
+    str = true 
+  else 
+    str = false 
+  end
+end 
+
+# User interface
+
+puts "How many employees will be processed?"
+count = gets.chomp.to_i 
+
+x = 0
+
+# Loop processing code until through every employee
+while x < count
+
+  puts "What is your name?"
+  name = gets.chomp
+  
+  puts "How old are you?"
+  # Converting string response to integer 
+  age = gets.chomp .to_i
+  
+  puts "What year were you born?"
+  # Converting string response to integer 
+  year = gets.chomp.to_i 
+  
+  # Converting birth year to age
+  age_2 = 2017 - year
```

Comment:
Using time can be pesky.  Because time changes we need to account for the future as well.  Check out the Ruby `Time` class and see what `Time.new.year` can do for your code.